packages/logging/src/writers/console.writer.ts

```diff
@@ -0,0 +1,101 @@
+import {inject, injectable} from "tsyringe";
+import {SeverityEnum} from "../enums/severity.enum";
+import {LogModel} from "../models/log.model";
+import {WriterInterface} from "../interfaces/writer.interface";
+import {Readable} from "stream";
+import * as util from "util";
+
+@injectable()
+export class ConsoleWriter implements WriterInterface {
+  public readableStream: Readable;
+  private stackedLogs: LogModel[] = [];
+
+  public constructor(@inject("%pristine.logging.numberOfStackedLogs%") private readonly numberOfStackedLogs: number,
+                     @inject("%pristine.logging.logSeverityLevelConfiguration%") private readonly logSeverityLevelConfiguration: number,
+                     @inject("%pristine.logging.logDebugDepthConfiguration%") private readonly logDebugDepthConfiguration: number,
+                     @inject("%pristine.logging.logInfoDepthConfiguration%") private readonly logInfoDepthConfiguration: number,
+                     @inject("%pristine.logging.logWarningDepthConfiguration%") private readonly logWarningDepthConfiguration: number,
+                     @inject("%pristine.logging.logErrorDepthConfiguration%") private readonly logErrorDepthConfiguration: number,
+                     @inject("%pristine.logging.logCriticalDepthConfiguration%") private readonly logCriticalDepthConfiguration: number,
+                     @inject("%pristine.logging.consoleWriterActivated%") private readonly consoleWriterActivated: boolean,
+                     ) {
+    this.initialize();
+  }
+
+  private initialize(){
+    if(this.isActive()) {
+      this.readableStream = new Readable({
+        objectMode: true,
+        read(size: number) {
+          return true;
+        }
+      });
+      this.readableStream.on('data', chunk => {
+        this.captureLog(chunk);
+      });
+    }
+  }
+
+  public isActive(): boolean {
+    return this.consoleWriterActivated;
+  }
+
+  private log(log: LogModel): void {
+    switch (log.severity) {
+      case SeverityEnum.Debug:
+        console.debug(log.message + " - Extra: " + util.inspect(log.extra, false, this.logDebugDepthConfiguration));
+        break;
+
+      case SeverityEnum.Info:
+        console.info(log.message + " - Extra: " + util.inspect(log.extra, false, this.logInfoDepthConfiguration));
+        break;
+
+      case SeverityEnum.Warning:
+        console.warn(log.message + " - Extra: " + util.inspect(log.extra, false, this.logWarningDepthConfiguration));
```

Comment:
Jpas sur de ce format là.

Reply:
Tu irais pour quoi ? Quelque chose comme 
console.warn(util.inspect({message: log.message, extra: log.extra}, false, this.logWarningDepthConfiguration));

Reply:
Je pense qu'on va devoir checker en ligne ou qu'est-ce que Winston fait pour output. Je pense que ya un genre d estandard qui peut être parsé facilement.